r/configure

```diff
@@ -201,7 +201,10 @@ else
 fi
 
 # Write to Makevars
-sed -e "s|@cflags@|$PKG_CFLAGS|" -e "s|@libs@|$PKG_LIBS|" src/Makevars.in > src/Makevars
+sed -e "s|@cflags@|$PKG_CFLAGS|" \
```

Comment:
Please revert this change and the one in Makevars.in. This will provoke the wrath of CRAN (and it should cause at least one nightly build failure).

Reply:
will do